app/main.py

```diff
@@ -1,19 +1,24 @@
-lucky_number = 777
-pi = 3.14
-one_is_a_prime_number = False
-name = "Richard"
+lucky_number = 777  # int
+pi = 3.14  # float
+one_is_a_prime_number = False  # bool
+name = "Richard"  # string
 my_favourite_films = [
     "The Shawshank Redemption",
     "The Lord of the Rings: The Return of the King",
     "Pulp Fiction",
     "The Good, the Bad and the Ugly",
     "The Matrix",
-]
-profile_info = ("michel", "michel@gmail.com", "12345678")
+]  # list
+profile_info = ("michel", "michel@gmail.com", "12345678")  # tuple
 marks = {
     "John": 4,
     "Sergio": 3,
+}  # dictionary
+collection_of_coins = {1, 2, 25}  # set
+
+sorted_variables = {
+    "mutable": [my_favourite_films, marks, collection_of_coins],
+    "immutable": [lucky_number, pi, one_is_a_prime_number, name, profile_info]
```

Comment:
You've sorted your variables based on mutability, which is great. However, you've classified `lucky_number`, `pi`, `one_is_a_prime_number`, `name` and `profile_info` as immutable, which is correct, but you forgot to include `pi` in the mutable list. Remember that floats are also immutable in Python.